templates/kotlin/src/main/kotlin/io/appwrite/Query.kt.twig

```diff
@@ -1,44 +1,157 @@
 package {{ sdk.namespace | caseDot }}
 
-class Query {
-  companion object {
-    fun equal(attribute: String, value: Any) = addQuery(attribute, "equal", value)
+/**
+ * Helper class for queries.
+ *
+ * **IMPORTANT** You can only query indexed attributes!
+ */
+@Suppress("NOTHING_TO_INLINE")
```

Comment:
With the `inline` modifier removed from the private methods, we can remove this warning suppression